Ginger/GingerCoreCommon/SelfHealingLib/SelfHealingConfig.cs

```diff
@@ -0,0 +1,46 @@
+﻿#region License
+/*
+Copyright © 2014-2021 European Support Limited
+
+Licensed under the Apache License, Version 2.0 (the "License")
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at 
+
+http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+#endregion
+using Amdocs.Ginger.Repository;
+
+namespace Amdocs.Ginger.Common.SelfHealingLib
+{
+    public class SelfHealingConfig : RepositoryItemBase
+    {
+        [IsSerializedForLocalRepository]
+        public bool EnableSelfHealing { get; set; }
+
+        [IsSerializedForLocalRepository]
+        public bool  PrioritizePOMLocator { get; set; }
+
+        [IsSerializedForLocalRepository]
+        public bool AutoFixAnalyzerIssue { get; set; }
+
+        [IsSerializedForLocalRepository]
+        public bool AutoUpdateApplicationModel { get; set; }
+
+        public bool SaveChangesInSourceControl { get; set; }
+
+        public bool RunFromAutomateTab { get; set; }
```

Comment:
we should not have this flag here. There is already existing flag for this. need to find and use the same flag for checking this